src/lib/tree/node.ts

```diff
@@ -101,4 +101,12 @@ export class MatNestedTreeNode<T> extends _MatNestedTreeNodeMixinBase<T>
 
     this.tabIndex = Number(tabIndex) || 0;
   }
+
+  ngAfterContentInit() {
+    super.ngAfterContentInit();
+  }
+
+  ngOnDestroy() {
+    super.ngOnDestroy();
+  }
```

Comment:
Is there a test that would have caught this?

Reply:
It only happen in aot or prod mode. We have test cover nested tree, but I'm not sure how can we run tests in aot mode.

Reply:
Hmm... something seems odd about this. Neither of those modes should affect the class construction, and the child classes should always inherit its base class's methods. Do you have a reproduction w/ aot?

Reply:
Done. Thanks for review